packages/backend/server/src/core/features/management.ts

```diff
@@ -116,9 +116,14 @@ export class FeatureManagementService {
     return this.feature.listFeatureWorkspaces(feature);
   }
 
+  // ======== User Feature ========
   async getUserFeatures(userId: string): Promise<FeatureType[]> {
-    return (await this.feature.getUserFeatures(userId)).map(
-      f => f.feature.name
-    );
+    const features = await this.feature.getUserFeatures(userId);
+    const types = features.filter(f => f.activated).map(f => f.feature.name);
+    // todo(@darkskygit): remove this after support cascade feature
+    if (types.includes(FeatureType.EarlyAccess)) {
+      types.push(FeatureType.Copilot);
```

Comment:
If we want to give them an AI subscription, I think this should be done by giving a stripe coupon instead of implementing it in the code.

Reply:
hack code, has been remove and add ea user migration